Src/BootCamp.Chapter/Program.cs

```diff
@@ -9,6 +11,118 @@ class Program
     {
         static void Main(string[] args)
         {
+            // Demo();
+
+            Console.Write(CalculateBMI(-100, 50));
+        }
+
+        public static void Demo()
+        {
+            for (int i = 0; i < 2; i++)
```

Comment:
Loop is fine, but since you need to call it twice, it would be slightly more easy to
```cs
public static void Demo()
{
  ProcessPersonBmi();
  ProcessPersonBmi();
}
```